### ISSUE TITLE
Support Rails 5.1.5

### ISSUE THREAD

**Issue by joshpencheon** [NHSDigital/activemodel-caution#2]
It has been released, and there are some `activemodel` changes we need to mirror.

https://github.com/rails/rails/compare/v5.1.4...v5.1.5

cc @miles-smith.

**Comment by joshpencheon** [comment]
```
$ git diff v5.1.4..v5.1.5 --stat -- activemodel
 activemodel/CHANGELOG.md                           |  6 +++
 activemodel/activemodel.gemspec                    |  5 +++
 activemodel/lib/active_model/gem_version.rb        |  2 +-
 .../lib/active_model/validations/callbacks.rb      | 22 ++++++-----
 .../test/cases/validations/callbacks_test.rb       | 43 ++++++++++++++++++++++
 5 files changed, 68 insertions(+), 10 deletions(-)
```

**Comment by joshpencheon** [comment]
Closed by 1226a5230ca49abd6db9.
